jde/modules/ROOT/pages/demo_poll_mbf_events.adoc

```diff
@@ -1,62 +1,72 @@
-= *PortX JD Edwards Connector for Mule 4 – Demo – Poll Transaction (MBF) Events*
+= PortX JD Edwards Connector for Mule 4 – Poll Transaction (MBF) Events
 :keywords: add_keywords_separated_by_commas
 :imagesdir: images
 :toc: macro
 :toclevels: 2
 
+////
 toc::[]
+////
```

Comment:
I don't think any of this is relevant for Antora - I don't think it hurts to have it, but it is extra clutter. I'm posted to the Guild to see if any of them know where it comes from, just to have a little more info.